lib/msal-browser/src/cache/BrowserCacheManager.ts

```diff
@@ -377,6 +377,61 @@ export class BrowserCacheManager extends CacheManager {
         this.internalStorage.setItem(key, JSON.stringify(entity));
     }
 
+    /**
+     * Gets the active account
+     */
+    getActiveAccount(): AccountInfo | null {
+        const activeAccountIdKey = this.generateCacheKey("active-account");
+        const activeAccountId = this.browserStorage.getItem(activeAccountIdKey);
+        return this.getAccountInfoByFilter({localAccountId: activeAccountId || ""})[0] || null;
+    }
+
+    /**
+     * Sets the active account's localAccountId in cache
+     * @param account 
+     */
+    setActiveAccount(account: AccountInfo | null): void {
+        const activeAccountIdKey = this.generateCacheKey("active-account");
```

Comment:
Is there an enum or constants file we can define this key?

Reply:
Done